light-zk.js/src/wallet/provider.ts

```diff
@@ -195,7 +206,7 @@ export class Provider {
     }
   }
 
-  private async loadPoseidon() {
+  async loadPoseidon() {
```

Comment:
I think we can remove this method, since it was primarily used in Provider.init and we replaced it there.